lib/stacks/eks-blueprint-stack.ts

```diff
@@ -132,8 +132,16 @@ export class BlueprintBuilder implements spi.AsyncStackBuilder {
         return this;
     }
 
-    public version(version: KubernetesVersion): this {
-        this.props = { ...this.props, ...{ version } };
+    public version(version: string | KubernetesVersion): this {
```

Comment:
Ideally, there will be one location in the codebase for the auto-selected version.

Reply:
@zjaco13 can we make this type `version: KubernetesVersion | "auto"`? This will take care of compile time check for correctness of the passed values.

Reply:
done